src/main/resources/swagger/swagger.sdk.json

```diff
@@ -0,0 +1,272 @@
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "$ref": "#\/definitions\/DidMappingRequest"
+            },
+            "in": "body",
+            "name": "didMapRequest",
+            "description": "didMapRequest",
+            "required": true
+          }
+        ],
+        "tags": [
+          "did-map-controller"
+        ],
+        "consumes": [
+          "application\/json;charset=UTF-8"
+        ]
+      }
+    },
+    "\/didmaps\/{applicationId}\/{userId}": {
+      "get": {
+        "summary": "Get a stored DID Map",
+        "deprecated": false,
+        "produces": [
+          "application\/json;charset=UTF-8"
+        ],
+        "description": "Retrieve a previously stored DID Map based of an App ID and User ID",
+        "operationId": "getDidMap",
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#\/definitions\/DidMappingResponse"
+            },
+            "description": "Did Map found."
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Did Map not found."
+          }
+        },
+        "parameters": [
+          {
+            "in": "path",
+            "name": "applicationId",
+            "description": "applicationId",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "path",
+            "name": "userId",
+            "description": "userId",
+            "type": "string",
+            "required": true
+          }
+        ],
+        "tags": [
+          "did-map-controller"
+        ]
+      }
+    }
+  },
+  "host": "localhost:57496",
+  "definitions": {
+    "DidMappingResponse": {
+      "type": "object",
+      "title": "DidMappingResponse",
+      "properties": {
+        "didMaps": {
+          "type": "array",
+          "items": {
+            "$ref": "#\/definitions\/DidMap"
+          }
+        }
+      }
+    },
+    "DidMappingRequest": {
+      "type": "object",
+      "title": "DidMappingRequest",
+      "properties": {
+        "didMaps": {
+          "type": "array",
+          "items": {
+            "$ref": "#\/definitions\/DidMap"
+          }
+        }
+      }
+    },
+    "DidMap": {
+      "type": "object",
+      "title": "DidMap",
+      "properties": {
+        "applicationId": {
+          "type": "string"
+        },
+        "didInfo": {
+          "$ref": "#\/definitions\/DidInfo"
+        },
+        "userId": {
+          "type": "string"
+        }
+      }
+    },
+    "ResponseEntity": {
```

Comment:
Hmzz, I don't think (read am sure), we want the ResponseEntity class ending up in the swagger file

Reply:
Yep that's an older bug. I've changed it to ResponseEntity<DidMappingResponse>. The regenerated the SDK but there were no significant changes. (New timestamps and ResponseEntity.java was taken out.)